mlflow/java/scoring/src/main/java/org/mlflow/sagemaker/MLeapPredictor.java

```diff
@@ -36,28 +36,18 @@
    * Constructs an {@link MLeapPredictor}
    *
    * @param modelDataPath The path to the serialized MLeap model
-   * @param inputSchemaPath The path to JSON-formatted file containing the input schema that the
-   *     model accepts
    */
-  public MLeapPredictor(String modelDataPath, String inputSchemaPath) {
+  public MLeapPredictor(String modelDataPath) {
```

Comment:
In order to avoid breaking changes since this is a public API, can we define an additional constructor that accepts the `inputSchemaPath` argument and reads the schema from the specified path? Can you also add javadoc to the legacy constructor indicating that it is deprecated?

Reply:
@dbczumar Sounds good, I've pushed the changes, added the previous constructor too and marked it as deprecated. Thank you!

Reply:
Awesome, thanks!